packages/data/src/world/mq/jabu_jabu_mq.yml

```diff
@@ -14,64 +14,180 @@
-    "MQ Jabu-Jabu Third Room East Chest": "can_use_slingshot && (is_child || (is_adult && can_dive_small))" #Future-proofing this for eventual ageless bow and slingshot. Adult cannot dive to the alcove without a scale.
-    "MQ Jabu-Jabu SoT Room Lower Chest": "(soul_ruto || can_play_elegy) && (is_child || (is_adult && can_dive_small))" #Future-proofing this for eventual ageless bow and slingshot. Adult cannot dive to the alcove without a scale.
-    "MQ Jabu-Jabu Boomerang Chest": "(soul_ruto || can_play_elegy) && soul_lizalfos_dinalfos && soul_like_like && soul_enemy(SOUL_ENEMY_STINGER)" #You will always have an acceptable weapon for this.
-    "MQ Jabu-Jabu GS SoT Block": "gs && (soul_ruto || can_play_elegy) && can_play_time && (is_child || (is_adult && can_dive_small))" #Future-proofing this for eventual ageless bow and slingshot. Adult cannot dive to the alcove without a scale.
-    "MQ Jabu-Jabu Cow": "can_play_epona && event(JABU_BIG_OCTO) && can_use_slingshot"
-    "MQ Jabu-Jabu Pot Underwater Alcove 1": "is_child || (is_adult && can_dive_small)" #Future-proofing this for eventual ageless bow and slingshot. Adult cannot dive to the alcove without a scale.
-    "MQ Jabu-Jabu Pot Underwater Alcove 2": "is_child || (is_adult && can_dive_small)" #Future-proofing this for eventual ageless bow and slingshot. Adult cannot dive to the alcove without a scale.
-    "MQ Jabu-Jabu Pot Boomerang Room 1": "(soul_ruto || can_play_elegy) && (is_child || (is_adult && can_dive_small))" #Future-proofing this for eventual ageless bow and slingshot. Adult cannot dive to the alcove without a scale.
-    "MQ Jabu-Jabu Pot Boomerang Room 2": "(soul_ruto || can_play_elegy) && (is_child || (is_adult && can_dive_small))" #Future-proofing this for eventual ageless bow and slingshot. Adult cannot dive to the alcove without a scale.
-    "MQ Jabu-Jabu Grass Main Room Top 1": "(is_child || (is_adult && can_dive_small)) && can_cut_grass && has_explosives_or_hammer"
-    "MQ Jabu-Jabu Grass Main Room Top 2": "(is_child || (is_adult && can_dive_small)) && can_cut_grass && has_explosives_or_hammer"
-    "MQ Jabu-Jabu Grass Main Room Bottom 1": "(is_child || (is_adult && can_dive_small)) && can_cut_grass"
-    "MQ Jabu-Jabu Grass Main Room Bottom 2": "(is_child || (is_adult && can_dive_small)) && can_cut_grass"
-    "MQ Jabu-Jabu Grass Main Room Bottom 3": "(is_child || (is_adult && can_dive_small)) && can_cut_grass"
-    "MQ Jabu-Jabu Grass Boomerang Room": "(is_child || (is_adult && can_dive_small)) && can_cut_grass"
-    "MQ Jabu-Jabu Grass Big Octo Top 1": "event(JABU_BIG_OCTO) && can_cut_grass"
-    "MQ Jabu-Jabu Grass Big Octo Top 2": "event(JABU_BIG_OCTO) && can_cut_grass"
-    "MQ Jabu-Jabu Grass Room After Big Octo": "event(JABU_BIG_OCTO) && can_cut_grass && can_use_slingshot"
+    "MQ Jabu-Jabu Wonder Item Elevator Cow": "can_use_slingshot"
```

Comment:
Does this fall in the water at all?

Reply:
Many wonderitems can do this. Ikna Graveyard ones can land on the trees, needing an entire scene reload.

Many ammo events are RNG what they give in any given instance.

DO we add it? Or just leave it up tot he RNG?

Reply:
i guess it's fine.